chain/gen/gen.go

```diff
@@ -468,10 +468,6 @@ func (cg *ChainGen) NextTipSetFromMinersWithMessagesAndNulls(base *types.TipSet,
 					return nil, xerrors.Errorf("making a block for next tipset failed: %w", err)
 				}
 
-				if err := cg.cs.PersistBlockHeaders(context.TODO(), fblk.Header); err != nil {
-					return nil, xerrors.Errorf("chainstore AddBlock: %w", err)
-				}
-
```

Comment:
What's the reason for dropping this?

Reply:
@arajasek -- I suspect this change is yours. I am going to add a call to PersistTipset here; I'm assuming that's what you wanted.

Reply:
Ah, it's already below. @magik6k -- we now have a PersistTipset method. This collects block headers into a tipset inside the loop, and then persists the tipset outside the loop.